build_runner_core/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: Core tools to organize the structure of a build and run Builders.
 repository: https://github.com/dart-lang/build/tree/master/build_runner_core
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
+  sdk: ">=2.18.0 <3.0.0"
```

Comment:
We should also update the version to `7.2.8-dev` here and mention the min sdk bump